tests/topotests/bgp_color_extcommunities/r1/bgpd.conf

```diff
@@ -0,0 +1,21 @@
+!
+router bgp 65001
+ bgp router-id 192.168.1.1
+ no bgp ebgp-requires-policy
+ no bgp network import-check
+ neighbor 192.168.1.2 remote-as external
+
+  network 10.10.10.10/24 route-map rmap
+  neighbor 192.168.1.2 route-map rmap out
+  neighbor 192.168.1.2 activate
+ exit-address-family
+!
+access-list al-any permit any
+
+
+route-map rmap permit 10
+ match ip address al-any
+ set extcommunity color 1
+ set extcommunity rt 80:987
+ set extcommunity color 100 55555 200
```

Comment:
color defined two times, intentionally?

Reply:
yes to verify that first one is not taken into account